.gitmodules

```diff
@@ -1,7 +1,9 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
-  branch = develop
+  #url = https://github.com/NOAA-EMC/fv3atm
+  #branch = develop
+  url = https://github.com/DeniseWorthen/fv3atm
+  branch = feature/noneg_zbot_export
```

Comment:
Is the fv3 branch updated? I see some changes in FV3/ccpp

Reply:
I fixed it now.